README.md

```diff
@@ -73,7 +73,7 @@ sudo rsyslog/install.sh -t TYPE -a TOKEN [--quite] [--filepath] [--filetag] [--a
 	- slow-log `/var/log/mysql/mysql-slow.log` (can be overrided using the option --slowlog)
 	- error `/var/log/mysql/error.log` (can be overrided using the option --errorlog)
 
-**-q | --quite** 
+**-q | --quiet** 
```

Comment:
I see that "quite" is being used in the usage example above, line 21, which needs update as well

Reply:
Very true, updated.